src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/ImportFAT2SectorStandardToPL.sql

```diff
@@ -0,0 +1,137 @@
+﻿
+CREATE PROCEDURE [dbo].[ImportFAT2SectorStandardToPL]
+(
+   @RunId int
+)
+AS
+-- ==========================================================================================================
+-- Author:      Sarma Evani
+-- Create Date: 17/Dec/2020
+-- Description: Import, Transform and Load FAT2 Framework Presentation Layer Table
+-- ==========================================================================================================
+BEGIN TRY
+		DECLARE @LogID int
+
+		/* Start Logging Execution */
+
+			  INSERT INTO Mgmt.Log_Execution_Results
+				  (
+					RunId
+				   ,StepNo
+				   ,StoredProcedureName
+				   ,StartDateTime
+				   ,Execution_Status
+				  )
+			  SELECT 
+					@RunId
+				   ,'Step-6'
+				   ,'ImportFAT2SectorStandardToPL'
+				   ,getdate()
+				   ,0
+
+			  SELECT @LogID=MAX(LogId) FROM Mgmt.Log_Execution_Results
+			  WHERE StoredProcedureName='ImportFAT2SectorStandardToPL'
+			  AND RunId=@RunID
+
+			BEGIN TRANSACTION
+
+					DELETE FROM [ASData_PL].[FAT2_StandardSector]
+
+					INSERT [ASData_PL].[FAT2_StandardSector]
+					(												
+						[Id],
+						[Title],
+						[Level],
+						[IntegratedDegree],
+						[OverviewOfRole],
+						[ApprenticeshipStandardId],
+						[SectorId],
+						[Sector],
+						[Keywords],
+						[TypicalJobTitles],
+						[StandardPageUrl],
+						[Version],
+						[RegulatedBody],
+						[Skills],
+						[Knowledge],
+						[Behaviours],
+						[Duties],
+						[CoreAndOptions]
+					)
+					select
+						std.[Id],
+						[Title],
+						[Level],
+						[IntegratedDegree],
+						[OverviewOfRole],
+						AppStandard.[StandardId],
+						[RouteId],
+						sector.[Route],
+						[Keywords],
+						[TypicalJobTitles],
+						[StandardPageUrl],
+						[Version],
+						[RegulatedBody],
+						[Skills],
+						[Knowledge],
+						[Behaviours],
+						[Duties],
+						[CoreAndOptions]
+					FROM [Stg].[FAT2_Standard] std JOIN [Stg].[FAT2_Sector] sector 
+					ON std.[RouteId] =  sector.[Id]	LEFT JOIN [ASData_PL].[Va_ApprenticeshipStandard] AppStandard
+					ON std.Id = AppStandard.LarsCode AND std.Title = AppStandard.StandardFullName
+
```

Comment:
Can we please make a join to Vacancies standard sector table and get a Reference Key similar to what you have done for Framework. Thanks